pycbc/types/config.py

```diff
@@ -513,6 +517,12 @@ def check_duplicate_options(self, section1, section2, raise_error=False):
                 "Section %s not present in ConfigParser." % (section2,)
             )
 
+        if section1.endswith("-defaultvalues") or \
+                section2.endswith("-defaultvalues"):
+            # Override the raise_error variable not to error when
+            # defaultvalues are given
+            raise_error = False
+
```

Comment:
I think there are some sanity checks to catch here.

The case:

```
[coinc]
option1 = 10

[coinc-defaultvalues]
option1=15
```

should not be allowed. The first means the second can never be used.